integration-tests/plugins/index.js

```diff
@@ -65,6 +65,7 @@ module.exports = (on) => {
         auth.stubUserMe(username, 12345, 'James Stuart', caseload),
         prisonApi.stubUserCaseloads(caseloads),
         tokenverification.stubVerifyToken(true),
+        keyworker.stubKeyworkerMigrated(),
```

Comment:
this was annoying me as was causing all signins to display the sorry page instead of the home page